common/metrics_evaluator.py

```diff
@@ -0,0 +1,47 @@
+#  @Date    : 2023-03-19
+#  @Editor  : Mikhail Kiselyov
+#  @E-mail  : kiselev.0353@gmail.com
+#  Provided as is
+
+import os
+from pathlib import Path
+import json
```

Comment:
Let's sort import in the following order: external and system `import`, external and system `from`,  local `import`, local `from`. Each group should be separated by a new line